prow/config.yaml

```diff
@@ -22,6 +22,15 @@ prowjob_namespace: default
 pod_namespace: test-pods
 log_level: info
 
+branch-protection:
+  Orgs:
```

Comment:
I know this is correct per the code, but the capitalization of Orgs and Repos looks odd in the configuration, as there is no other key in this file that begins with a capital letter.

Reply:
Oh good catch! Actually I think it is supposed to be wrong per the code?

https://github.com/kubernetes/test-infra/blob/master/prow/config/branch_protection.go

Reply:
You are absolutely right! I was looking at the wrong place.